mex.go

```diff
@@ -390,7 +390,7 @@ func (mexset *messageExchangeSet) removeExchange(msgID uint32) {
 // will write to the send channel.
 func (mexset *messageExchangeSet) expireExchange(msgID uint32) {
 	mexset.log.Debugf(
-		"Removing %s message exchange %d due to timeout or cancelation",
+		"Removing %s message exchange %d due to timeout, cancelation or blackhole",
```

Comment:
cancellation*

Reply:
Indeed. Corroboration for consistency (since technically both spellings are accepted):

```
tchannel-go ❯ git grep -i cancellation
CHANGELOG.md:* Fix context cancellation not cancelling Dial attempts (#541)
inbound.go:     // TODO(prashant): This is an expensive way to check for cancellation. Use a heap for timeouts.
inbound.go:                             call.log.Debugf("Wait for timeout/cancellation interrupted by error: %v", call.mex.errCh.err)
mex.go:// It returns any existing errors (timeout, cancellation, connection errors).
mex.go: // 1. Timeouts/cancellation (mex.ctx errors)
mex.go: // 1. Timeouts/cancellation (mex.ctx errors)

tchannel-go ❯ git grep -i cancelation
mex.go:         "Removing %s message exchange %d due to timeout or cancelation",
```